### DISCUSSION TITLE
Identification of Segregation Sites and QTLs

### DISCUSSION THREAD

**Discussion by sandeepgowdan** [gaynorr/AlphaSimR#181] [Q&A]
How do you recommend identifying the minimum number of segregation sites and determining the number of QTLs controlling a trait for simulation purposes?

**Comment by gregorgorjanc** [comment]
This really depends on what you need. Ideally, we would implement the best species genome and demographic model parameters for the species, which will always give you plenty of polymorphic (segregating) sites, from which we choose desired number of QTL and markers for other work, such as GWAS or genomic prediction studies. How many QTL you need is a tricky question, since we often do not know this for many traits, but many traits seem to be quite complex and polygenic, so "many" (say 100-1000 per chromosome) is often advised, but it really depends what you are after.

**Comment by gregorgorjanc** [comment]
Just to add that nQTL is important parameter for dominance simulations! @gaynorr has worked out some very useful guidelines in that space. We are working on a manuscript describing this aspect.
